software/docmost.yml

```
@@ -0,0 +1,11 @@
+name: "docmost"
+website_url: "https://docmost.com/"
+source_code_url: "https://github.com/docmost/docmost"
+description: "Docmost is an open-source collaborative wiki and documentation software. It is an open-source alternative to Confluence and Notion."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
```

[Install guide](https://docmost.com/docs/installation) shows main installation method to be docker

```suggestion
platforms:
  - Docker
  - Nodejs
```